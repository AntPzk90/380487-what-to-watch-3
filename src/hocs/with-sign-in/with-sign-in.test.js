import React from 'react';
import renderer from 'react-test-renderer';
import withSignIn from './with-sign-in.jsx';


it(`SnapshotTest withSignIn`, () => {

  const tree = renderer
  .create(
      <withSignIn
        onSendForm={() => {}}
        onSubmitBtnClick={() => {}}
        isEmailInputValid={true}
        loginRef={() => {}}
        password={() => {}}
      />
  )
  .toJSON();

  expect(tree).toMatchSnapshot();
});
