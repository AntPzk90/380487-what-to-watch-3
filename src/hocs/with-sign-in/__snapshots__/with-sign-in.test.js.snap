// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SnapshotTest withSignIn 1`] = `
<withSignIn
  isEmailInputValid={true}
  loginRef={[Function]}
  onSendForm={[Function]}
  onSubmitBtnClick={[Function]}
  password={[Function]}
/>
`;
